BAZProject/Views/MovieDetail/CollectionViewCells/SimilarsCollectionViewCell.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  SimilarsCollectionView.swift
+//  BAZProject
+//
+//  Created by Mario Arceo on 01/03/23.
+//
+
+import UIKit
+
+class SimilarsCollectionViewCell: UICollectionViewCell {
+    
+    let identifier = "SimilarsCustomCell"
```

Comment:
Para no crear una instancia de esta clase es mejor poner el identifier con constante estática.
```suggestion
    static let identifier = "SimilarsCustomCell"
```